### ISSUE TITLE
Error, Unknown substitution key ' matrix.gap-branch '

### ISSUE THREAD

**Issue by ssiccha** [gap-system/PackageMaker#48]
When I run `PackageWizard();` and ask it to prepare the package for github, then I get the following error:
```
Error, Unknown substitution key ' matrix.gap-branch '
 at /home/sergio/projects/pkg/PackageMaker/gap/PackageMaker.gi:59 called from
TranslateTemplate( fail, ".github/workflows/CI.yml", pkginfo ); at /home/sergio/projects/pkg/PackageMaker/gap/PackageMaker.gi:699 called from
<function "PackageWizard">( <arguments> )
```
Looks, like it is somehow trying to substitute the part `name: ${{ matrix.gap-branch }}` in the `CI.yml`.

**Comment by juniors90** [comment]
"chmod" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
Error, Unknown substitution key ' matrix.gap-branch '
 at /proc/cygdrive/C/gap-4.10.2/pkg/PackageMaker/gap/PackageMaker.gi:59 called from
TranslateTemplate( fail, ".github/workflows/CI.yml", pkginfo
 ); at /proc/cygdrive/C/gap-4.10.2/pkg/PackageMaker/gap/PackageMaker.gi:699 called from
<function "PackageWizard">( <arguments> )
 called from read-eval loop at *stdin*:7
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue

= (

**Comment by fingolfin** [comment]
Thanks for the reports. Fixed by 4b2dc37c7172c0e9afdcef955580df0452f6034b
